src/Core/Models/Business/OrganizationLicense.cs

```diff
@@ -137,6 +138,7 @@ public OrganizationLicense()
     public int? SmSeats { get; set; }
     public int? SmServiceAccounts { get; set; }
     public bool LimitCollectionCreationDeletion { get; set; } = true;
+    public bool AllowAdminAccessToAllCollectionItems { get; set; }
```

Comment:
Needs to default to `true`. This covers the case where a client loads an old license file which doesn't have this value.